airflow/assets/manager.py

```diff
@@ -273,7 +273,7 @@ def _postgres_send_dag_priority_parsing_request(cls, file_locs: Iterable[str], s
         from sqlalchemy.dialects.postgresql import insert
 
         stmt = insert(DagPriorityParsingRequest).on_conflict_do_nothing()
-        session.execute(stmt, {"fileloc": fileloc for fileloc in file_locs})
+        session.execute(stmt, [{"fileloc": fileloc} for fileloc in file_locs])
```

Comment:
Is it possible to test it?

Reply:
I'm not sure whether there's a way to add a meaningful test. 🤔 But I'll take some time and try it today! Thanks!